kyaml/fn/framework/frameworktestutil/frameworktestutil.go

```diff
@@ -83,20 +86,19 @@ func (rc ResultsChecker) Assert(t *testing.T) bool {
 		return nil
 	})
 
+	require.NotZero(t, rc.testsCasesRun, "No complete test cases found in %s", rc.TestDataDirectory)
+
 	return true
 }
 
-func (rc ResultsChecker) compare(t *testing.T, path string) {
+func (rc *CommandResultsChecker) compare(t *testing.T, path string) {
 	// cd into the directory so we can test functions that refer
 	// local files by relative paths
 	d, err := os.Getwd()
-	if !assert.NoError(t, err) {
-		t.FailNow()
-	}
+	require.NoError(t, err)
+
 	defer func() { _ = os.Chdir(d) }()
```

Comment:
I would actually use `defer func() { require.NoError(t, os.Chdir(d) } ()` that way you get notified if this fails